### ISSUE TITLE
Error when pasting with adv dupe 2

### ISSUE THREAD

**Issue by tomekb530** [Weol/WUMA#75]
![image](https://user-images.githubusercontent.com/8088960/166985706-653c2b65-3682-4bc7-93ec-240fe7e313d2.png)

what i  quickly found in code is the AddCount function is called with nil as first parameter and then the  WUMA.ChangeTotalLimits is called with nil and it causes errors

**Comment by Weol** [comment]
I'm investigating this.

**Comment by storm37000** [comment]
It seems to happen when somebody attempts to spawn any wiremod entity that has any entry in WUMA, even if their team isnt the one with the entry.

**Comment by storm37000** [comment]
This addon fixes this vanilla game bug.
https://steamcommunity.com/sharedfiles/filedetails/?id=2637764673

**Comment by wrefgtzweve** [comment]
Having this issue too, only started happening after i updated to this commit https://github.com/Weol/wuma/commit/d15667c3fe53ea9e2662bdd3adbaaa088d6d07f0

**Comment by wrefgtzweve** [comment]
Yeah, after i used an older version I'm not longer having this issue. Just use an older version till this is fixed i guess.

**Comment by gmodmurderserver** [comment]
this breaks when limits are infinite aswell.

**Comment by deathking61** [comment]
[AdvDupe2Notify]    addons/wuma-master/lua/wuma/limits.lua:303: table index is nil[AdvDupe2Notify]    addons/wuma-master/lua/wuma/limits.lua:303: table index is nil


when will this be fixed?

**Comment by VoLKov16t8** [comment]
probably never, the author just ignore the issue LOL

**Comment by deathking61** [comment]
well thats a crock of shit. if the dev cant fix it fine but refusing to do
so is pathetic
